### PR TITLE
[iOS] Sample App Update

### PR DESCRIPTION
**Pull request by jwoo-msft**
## Description
* Allow flexible width for AdaptiveCards when the width is set to 0
* Fixed background image issue when loaded from a resource bundle
* Fixed having extra space when a large image is scaled down in a column.
* Updated Input Wrapper View's intrinsic content size
* Update Content View to better manage its subviews content size
* Fixed background Image rendering for inner AdaptiveCards of ShowCard
* Changed sample app to update a row instead of table view when resources are loaded
* Updated sync mechanism
* Updated scroll behavior
* Updated app to test images in a resource bundle

### **Before Fix**
Screen Shot 2021-03-05 at 5 13 14 PM
Screen Shot 2021-03-05 at 5 07 12 PM

### **After Fix**
Screen Shot 2021-03-05 at 1 28 53 PM
Screen Shot 2021-03-05 at 1 20 00 PM

### CODE REVIEW

File: source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m [hunk 1]
```diff
@@ -20,12 +20,17 @@
...
 
 @interface ViewController () {
     BOOL _enableCustomRenderer;
     id<ACRIBaseActionSetRenderer> _defaultRenderer;
     ACRChatWindow *_dataSource;
+    dispatch_queue_t _global_queue;
+    dispatch_group_t _async_tasks_group;
+    BOOL _ran;
+    BOOL cardRendered;
+    BOOL cardLoaded;
```
Comment (paulcam206) [inline source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m:33]: should these be

```suggestion
    BOOL _cardRendered;
    BOOL _cardLoaded;
```

or is there some ObjC stuff going on here? :)

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.mm [hunk 2]
```diff
@@ -94,27 +95,95 @@ void renderBackgroundImage(const std::shared_ptr<AdaptiveCards::BackgroundImage>
...
+// apply contraints for 'Cover' fill mode
+// the backgroundView is set on the targetView
+void renderBackgroundCoverMode(UIView *backgroundView, ACRContentStackView *targetView)
+{
+    if (![backgroundView isKindOfClass:[UIImageView class]] || targetView.isBackgroundImageSet) {
+        return;
+    }
+    
+    UIImageView *imageView = (UIImageView *)backgroundView;
+    UIImage *image = imageView.image;
```
Comment (paulcam206) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.mm:132]: safe to assume `imageView` is not null?

**Review by paulcam206 — APPROVED** [review]
Looks good -- just to be clear, there aren't any breaking changes here?

**Comment by jwoo-msft** [comment]
> Looks good -- just to be clear, there aren't any breaking changes here?

Yes, there are no breaking changes.

**Pull request merged**